koans/about_dice_project.rb

```diff
@@ -3,7 +3,19 @@
 # Implement a DiceSet Class here:
 #
 # class DiceSet
-#   code ...
+class DiceSet
+
+  attr_reader :values
+
+  def roll(numDie)
+
+    @values = []
+
+    numDie.downto(1) { |i|
+      @values.push(1 + rand(6))
```

Comment:
@values is needless. Use to_a method to solve this. For example (1..2).to_a give you [1,2]